lib/decorators-callbacks.js

```diff
@@ -18,11 +18,24 @@ module.exports.decorateLambdaReqCallback = function(getPrincipalId) {
   };
 };
 
-module.exports.decorateAddCORSCallback = function() {
+module.exports.decorateAddCORSCallback = function(corsOpts, isOptionMethod) {
+  // TODO: get default cors options from variables
+  const cors = {
+    origins: ['*'],
+    methods: ['GET', 'PUT', 'HEAD', 'PATCH', 'POST', 'DELETE', 'OPTIONS'],
+    headers: ['Authorization', 'Content-Type', 'x-amz-date', 'x-amz-security-token']
+  };
+  if(corsOpts instanceof Object) {
+    Object.assign(cors, corsOpts);
+  } else if(corsOpts != true) {
```

Comment:
this should never be the case, we could remove it (or in the worst case always use `!==`, I should add it to the linter)